docs/guidelines/content-guidelines/05-tabs.md

```diff
@@ -0,0 +1,237 @@
+---
+title: Tabs & toggles
+---
+
+Tabs and toggles allow you to render several versions of a given section in one document or have several versions of one document on `http://kyma-project.io`.
+
+You can use tabs and/or documentation toggle to include instructions for different operating systems (OS) in one guide or to describe installation guides that differ only in a few steps.
+
```

Comment:
Can we add a sentence that explains what's the main difference between tabs and toggles (iirc, tabs are docsify-specific markdown, whereas toggles use html within markdown?), and maybe a screenshot how they look rendered? Just as a first pointer to support the decision what to use in a specific case.